crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -267,6 +280,28 @@ fn get_card_request_data(
     )))
 }
 
+fn get_debtor_info(
+    item: &types::PaymentsAuthorizeRouterData,
+) -> CustomResult<Option<DebtorInformation>, errors::ConnectorError> {
+    let payment_method_type = item
+        .request
+        .payment_method_type
+        .as_ref()
+        .ok_or(errors::ConnectorError::MissingPaymentMethodType)?;
+    match payment_method_type {
+        enums::PaymentMethodType::Blik => Ok(Some(DebtorInformation {
+            name: item
+                .get_billing()?
+                .address
+                .as_ref()
+                .map(|address| address.first_name.clone().unwrap_or_default())
```

Comment:
why are we using `unwrap_or_default` here? And is it intended to pass billing_address name in DebtorInformation?